### PR TITLE
Add wordlist for RCE of length > 3 as 932235

### PR DESCRIPTION
**Pull request by theseion**
The refactoring of 932100 and 932105 did not consider attacks *without* evasion attempts. This commit adds a new rule, 932235, that detects attacks using the word lists that were previously used by 932100 and 932105 (combined since the rule length limit for httpd no longer applies).

### CODE REVIEW

**Comment by fzipi** [comment]
Excellent. Will review later today. It is in sync with unix-shell, right?

**Comment by theseion** [comment]
I haven't checked yet. I just used the last version of 932100 and 932105. I'll check and let you know.

**Review by fzipi — COMMENTED** [review]
Waiting on confirmation still.

**Comment by theseion** [comment]
@fzipi checked.

File: regex-assembly/932235.ra [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  ##! quoting prefix
+  (?:
+  ##!=>
+
+  ##! 'ifconfig'
+  '
+  ##! "ifconfig"
+  \"
+  ##!=>
```
[the remark this answers is not in the document]
Comment (theseion) [reply regex-assembly/932235.ra:81]: Yes. I have that as a separate issue. The problem is that there are subtle differences between the files, so we need to be careful.

[the remark this answers is not in the document]
Comment (fzipi) [reply regex-assembly/932235.ra:81]: That was my fear. Then this looks good, and let's follow up in another issue.

**Review by fzipi — APPROVED** [review]

**Pull request merged**